### DISCUSSION TITLE
Can we already start using the new "Assignment Repositories and Starter Code"?

### DISCUSSION THREAD

**Discussion by taiya** [community/community#91543] [GitHub Education] [answered]
### Hi everyone,

> Assignment acceptance and starter code repositories will be changing soon. [Review the changes and prepare your assignments.](https://github.blog/changelog/2023-12-08-upcoming-changes-to-assignment-repositories-and-starter-code-in-github-classroom/)

Can we start trying this out, or there is no beta program?
This is quite useful, as students tend to clone as soon as they see a link.
Also useful for material that was not extensively tested, as it allows top-down corrections.

Thanks

**Comment by SirYancy** [comment]
I would also really like to know. This was announced months ago, the banner says that the changes will take effect in January. And now it's almost not January anymore. Can we please get an update on this so that we can make appropriate preparations? I'm trying to teach classes here!

**Comment by RyanHecht** [comment, marked as answer]
Hi @taiya and @SirYancy! The Public Beta for this feature has launched! You can opt-in each of your Classrooms from their Dashboards! Here's the Changelog post with more info: https://github.blog/changelog/2024-01-22-migrating-github-classroom-assignment-repository-creation-from-create-from-template-to-forks-public-beta/

**Reply by SirYancy** [reply]
Hi! Will opting in affect our existing assignments?

**Reply by RyanHecht** [reply]
If you opt a classroom into the beta, existing student assignment repositories will not be affected. Any NEW acceptances of existing assignments will be forks, and their starter code repos will be subject to the new requirements!
